src/main/java/com/razorpay/VirtualAccountClient.java

```diff
@@ -31,9 +31,7 @@ public VirtualAccount edit(String id, JSONObject request) throws RazorpayExcepti
   }
 
   public VirtualAccount close(String id) throws RazorpayException {
-    JSONObject request = new JSONObject();
-    request.put("status", "closed");
-    return patch(String.format(Constants.VIRTUAL_ACCOUNT_EDIT, id), request);
+    return post(String.format(Constants.VIRTUAL_ACCOUNT_CLOSE, id));
```

Comment:
wont it be request.post?

Reply:
no, request was a JsonObject's instance, this post is http method POST